libtiledbsoma/src/CMakeLists.txt

```diff
@@ -1,4 +1,9 @@
-message(STATUS "Starting TileDB-SC build.")
+if(DEFINED ENV{LIBTILEDBSOMA_DEBUG_BUILD})
+  message(STATUS "")
+  message(STATUS "START libtiledbsoma/src/CMakeLists.txt")
+else()
+  message(STATUS "Starting TileDB-SC build.")
```

Comment:
LGTM. 

Minor request: Could you change this to `TileDB-SOMA`? We missed this one in the conversion to SOMA.

Reply:
dangit i was gonna do just that but sneezed & forgot :)